rust-runtime/aws-smithy-http-server-python/src/types.rs

```diff
@@ -6,19 +6,20 @@
 //! Python wrapped types from aws-smithy-types and aws-smithy-http.
 
 use std::{
+    future::Future,
     pin::Pin,
     sync::Arc,
     task::{Context, Poll},
 };
 
 use bytes::Bytes;
-use parking_lot::Mutex;
```

Comment:
I think you can get rid of the `parking_lot` dependency entirely, no?

Reply:
I guess not 😞 

https://github.com/awslabs/smithy-rs/blob/30d985ad9d39849bb40705d244413eae2764ba7f/rust-runtime/aws-smithy-http-server-python/src/server.rs#L10